PuzzleSlider/src/Engine/GameData.java

```diff
@@ -40,6 +44,10 @@ public String toString() {
 			out.append( Arrays.toString(mTiles[i]));
 			out.append(System.lineSeparator());
 		}
+		out.append("Pocet tahov: " + mMoveCount);
```

Comment:
Didnt put much thougth to which language im using, since it was added just for testing purposes. Additionally our application is going to have non-console design, so these contents of toString() fuction will be irrelevant at the end of the sprint.
To the second point. I dont really know. I've been using that long time and didnt see any problem there.
I can rework that if you insist, but i think it's irrelevant.